docs/use_cases/model_applications/tc_and_extra_tc/UserScript_ASCII2NC_PointStat_fcstHAFS_obsFRD_NetCDF.py

```diff
@@ -0,0 +1,166 @@
+"""
+Point-Stat: Standard Verification for CONUS Surface 
+==============================================================================
+
+model_applications/tc_and_extra_tc/UserScript_ASCII2NC_PointStat_fcstHAFS_obsFRD
+_NetCDF.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+# To provide useful statistical information on the relationship between observation data
+# in point format to a gridded forecast. These values can be used to assess the skill 
+# of the prediction. Statistics are store as partial sums to save space and Stat-Analysis
+# must be used to compute Continuous statistics.
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Forecast:** HAFS temperature
+# | **Observation:** HRD Dropsonde data 
+#
+# | **Location of Model forecast and Dropsonde files:** All of the input data required for this use case can be found in the sample data tarball. Click here to download: https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data
```

Comment:
```suggestion
# | **Location of Model forecast and Dropsonde files:** All of the input data required for this use case can be found in the sample data tarball. Click `here <https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data>`_ to download.
```
Suggesting moving and reformatting the link so that user's can click on the word "here" to go to the link.